### PR TITLE
docs(java outbox): Improve description for `.cdsrc.json`

### PR DESCRIPTION
**Pull request by bugwelle**
While the text mentions `.cdsrc.json`, it could just as well show the file using the code-group feature.

This commit does exactly that.

### CODE REVIEW

File: java/outbox.md [hunk 1]
```diff
@@ -34,22 +34,35 @@ Once the transaction succeeds, the messages are read from the database table and
 
 To enable the persistence for the outbox, you need to add the service `outbox` of kind `persistent-outbox` to the `cds.requires` section in the _package.json_ or _cdsrc.json_. Please note that the _cdsrc.json_ file represents already the `cds` section and only the `requires` section should be added to the _cdsrc.json_ file:
```
Comment (renejeglinsky) [inline java/outbox.md:35]: I just recently learned that the _.cdsrc.json_ can also hold a cds section. So we don't need to differentiate and keep it simpler.
```suggestion
To enable the persistence for the outbox, you need to add the service `outbox` of kind `persistent-outbox` to the `cds.requires` section in the _package.json_ or _cdsrc.json_:
```

Comment (bugwelle) [reply java/outbox.md:35]: It can? Great!

Comment (renejeglinsky) [reply java/outbox.md]: @chgeo My statement that _.cdsrc.json_ can hold a `cds` section is correct, or?

Comment (chgeo) [reply java/outbox.md]: Yes, nowadays you also can have such a section in `.cdsrc.json` while traditionally you couldn't.

Comment (renejeglinsky) [reply java/outbox.md]: Thanks! Then I'll remove the code-groups here as we don't need them anymore.

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**